### DISCUSSION TITLE
懒人包2024V2

### DISCUSSION THREAD

**Discussion by hooke007** [hooke007/mpv_PlayKit#495] [mpv 通用]
## 公共维基计划

文档站新增一个公共wiki计划。
官方站不可能允许本地化，按alpha排序也看起来杂乱无章，对新手来说看似大海捞针，所以自建一个开放编辑权限的wiki。
见 https://github.com/hooke007/mpv_doc-CN/wiki/1_%E8%84%9A%E6%9C%AC

_过于简单的脚本（能用profile实现的，比如播放时自动置顶）就别贴了_

## 其它注意点

- RTX VSR 已原生支持，见 https://github.com/hooke007/MPV_lazy/discussions/286#discussioncomment-5252792
- intel vsr 也已支持，本人无intel cpu，请自行测试 `--d3d11vpp=scale=2:scaling-mode=intel` 此参数注意点同上
- RTX HDR 也已原生支持，需要自行更新mpv本体至此提交 https://github.com/mpv-player/mpv/commit/6ca3752f758c1e5452f90348993fc2b2653aaa2b 之后的版本 https://github.com/hooke007/MPV_lazy/discussions/286#discussioncomment-9718855

## 主要问题追踪

- [ ] 文档站会和下个年终版本一起更新
- [x] 2024.8.27 更新本体以修复d3d11vpp滤镜的排序兼容问题
- [x] 2024.9.2 更新本体以修复d3d11vpp滤镜的片源兼容性
- [x] 2024.9.4 更新本体以修复在vo_gpu下启用 `--interpolation` 时的画面冻结问题
- [ ] `--deinterlace` 在d3d11va纯硬解下会失效，如有需求请自行更新本体至此提交 https://github.com/mpv-player/mpv/commit/f6d931301bd226239d95f59e3f18abfeab692ee9 之后的版本。否则自行切换其它解码API
- [ ] 有个特定hevc编码参数导致的解码卡顿问题 https://github.com/mpv-player/mpv/issues/14806
 2024.9.14 已由FFmpeg修复，如有需求自行更新mpv本体
- [ ] 有个似乎是编译工具链产生的跳转卡顿问题 https://github.com/shinchiro/mpv-winbuild-cmake/issues/705
 **暂时没有彻底的解决方法，目前只能取消掉其中任意一个参数。**

<hr /><em>This discussion was created from the release <a href='https://github.com/hooke007/MPV_lazy/releases/tag/20240824'>mpv-lazy-2024V2</a>.</em>

上期 https://github.com/hooke007/MPV_lazy/discussions/442

**Comment by SeanAshe** [comment]
为什么我界面是英文的
<img width=100 src=https://github.com/user-attachments/assets/04d73442-46e0-4e9d-a1c2-293bd1d95bf4> <img width=100 src=https://github.com/user-attachments/assets/6a5e99a5-3438-400e-8912-fff182077113>

**Reply by SeanAshe** [reply]
<img width=100 src=https://github.com/user-attachments/assets/4bf29f3b-cea4-4cb8-ba22-d355f5b03a00>
我把所有的slang都改成zh-cn也不起作用

**Reply by hooke007** [reply]
你下了什么东西
https://github.com/hooke007/MPV_lazy/blob/main/portable_config/script-opts/uosc_lang.conf
这个文件我都没打包进去，你是怎么凭空下进去的
判断为你下错东西了，下载仓库直接复制到目录不是正确的操作。
https://github.com/hooke007/MPV_lazy/wiki#%E5%8F%91%E8%A1%8C

**Reply by SeanAshe** [reply]
我从sorce code里复制的

**Reply by hooke007** [reply]
你要复制仓库的东西可以先看一下说明每个文件是做什么的

**Reply by SeanAshe** [reply]
包里的portable_config 只有一个vs目录

**Reply by SeanAshe** [reply]
其他的怎么补全

**Reply by hooke007** [reply]
什么怎么补全，下标准包了吗，vsNV不是写了是补丁包吗。。。难道还要我翻译什么是补丁吗

**Comment by signin404** [comment]
新版MPV不知道改了什么
display-resample性能比v0.37.0-754-g58eae25f差

**Reply by hooke007** [reply]
我no-config测下来差异只有误差级别

**Reply by signin404** [reply]
开VS把显卡跑到95%以上比较明显
我测试同一片源跑2分钟会多掉10-20帧

**Reply by hooke007** [reply]
这复现条件太苛刻了而且带入了外部程序，很难说是mpv的问题

**Reply by signin404** [reply]
怀疑是d3b251b导致的
df166c1正常
d2bd77a性能下降

**Reply by hooke007** [reply]
你只要能找到简单的内部参数进行量化复现的话我就能报告。。。

**Reply by signin404** [reply]
加速播放也能复现
2.6倍速跑10分钟

g58eae25f
output: 11 Mistimed: 13 Delayed: 0
gecc6cdde
output: 76 Mistimed: 112 Delayed: 34

还发现VSR+RIFE+display-resample同时开
即使显卡最高睿频占用率40%也会狂掉帧

**Reply by hooke007** [reply]
我这里4倍速拉了十分钟除了暂停那一下，几乎0掉帧增长
和硬件本身性能关联太大了，这个测试没有普适性

**Reply by signin404** [reply]
profile匹配当前使用的硬件解码要怎么写?
试了几个都无效

p["hwdec-current"]==nvdec-copy
p["hwdec-current"]==nvdec_copy
p["hwdec_current"]==nvdec-copy
p["hwdec_current"]==nvdec_copy
hwdec-current==nvdec-copy
hwdec-current==nvdec_copy
hwdec_current==nvdec-copy
hwdec_current==nvdec_copy

同一个profile里有延迟执行功能么?

**Reply by hooke007** [reply]
匹配的对象类型是字符串，你写错了

`p["hwdec-current"] == "nvdec-copy"`

不延迟

**Comment by darkblue112233** [comment]
刚看了看配置文档，大佬改的真细，我都半年没更新了，一直用的anime4k官方给的配置方案A+A和RIFE二倍帧

**Comment by Asendaushoe** [comment]
自动开启着色器应该在哪里写配置文件，我写在mpv.conf里没有生效

**Reply by hooke007** [reply]
先去去看看必读说明

**Reply by Asendaushoe** [reply]
指个路吧，不知道说的是哪个，我读过的都没解决问题

**Reply by hooke007** [reply]
![image](https://github.com/user-attachments/assets/3100053a-3378-4838-be85-8fd9cb84bbc3)

为了防止有的人看不懂，我还特意mpv.conf写了，我也不知道是什么原因这个文件里就一句中文还能被无视
![image](https://github.com/user-attachments/assets/de261e53-edcf-47b5-9e97-b8cb8199cfb4)

**Comment by SuperBlackWednesday** [comment]
这个版本杜比视界 hdr是需要自己开启的吗？

**Reply by hooke007** [reply]
你是在说dv hdr直出？看faq，一直都是需要手动设置的

**Comment by xiaoyutx94** [comment]
怎么打开网址呢？

**Reply by hooke007** [reply]
这个版本直接拖进去。

**Reply by sudsb** [reply]
怎么拖呢，现在我想播放在线的视频，视频链接已经有了，现在找不到播放在线网址的入口了。

**Reply by hooke007** [reply]
鼠标拖进窗口啊，难道还要我录个屏怎么拖文字进窗口吗。。。

**Reply by sudsb** [reply]
> 鼠标拖进窗口啊，难道还要我录个屏怎么拖文字进窗口吗。。。

但是我链接没办法拖过去，是不是不支持这种视频呢。地址是
https://source.qdexam.com/UploadVideo/GongChengLei/ZhuCeAnQuanGongChengShi/ShiPinKeCheng/JianZhuShiGongAnQuan/JiChuBan/2023jzsgaqjcb08.mp4

**Reply by hooke007** [reply]
不能复现，我这能拖。拖不了就控制台手敲命令
```
loadfile ``https://xxxxxxxxx``
```

**Reply by sudsb** [reply]
> 不能复现，我这能拖。拖不了就控制台手敲命令
> 
> ```
> loadfile ``https://xxxxxxxxx``
> ```
ok，非常感谢

**Reply by leezm** [reply]
![image](https://github.com/user-attachments/assets/85f2f8a3-7f56-4623-acba-8f42c8072637)
我下的mpv-lazy-20240824.exe标准版,win11系统,打开后界面上没有提示拖拽文件或者URL到此,手动拖放文件显示🚫符号,这个需要单独设置吗?

**Reply by hooke007** [reply]
本来就没什么提示。那句话是原始osc的固定文本。

没遇到过阻止拖放

**Reply by leezm** [reply]
> 本来就没什么提示。那句话是原始osc的固定文本。
> 
> 没遇到过阻止拖放

![PixPin_2024-10-06_11-33-25](https://github.com/user-attachments/assets/14f7a3cf-1b7e-433b-a433-8ec74babac05)
我从资源管理器里拖放一个MP4文件就这样,显示🚫

**Reply by hooke007** [reply]
我知道你在说什么。权限问题 https://github.com/mpv-player/mpv/issues/9003
假定你使用了管理员权限

**Reply by leezm** [reply]
> 我知道你在说什么。权限问题 [mpv-player/mpv#9003](https://github.com/mpv-player/mpv/issues/9003) 假定你使用了管理员权限

嗯嗯,试了下可以了,谢谢!我还怕它权限不够所以一开始设置了默认管理员权限运行...

**Reply by hooke007** [reply]
一个播放器要逆天的管理员权限做什么。
你这软件习惯也是简直了。。。莫名其妙给不认识的软件管理员当肉鸡吗

**Reply by leezm** [reply]
> 一个播放器要逆天的管理员权限做什么。 你这软件习惯也是简直了。。。莫名其妙给不认识的软件管理员当肉鸡吗

这软件来路很正啊~

**Reply by hooke007** [reply]
编译是我自己搞的，封装的东西你也没去查，mpv跑个旁路自动化脚本毫无门槛，这么快的结论不是纯扯淡吗

**Reply by hooke007** [reply]
mpv给第三方的下载地址都写免责声明呢，你这自信是哪来的。。。

**Reply by leezm** [reply]
> 编译是我自己搞的，封装的东西你也没去查，mpv跑个旁路自动化脚本毫无门槛，这么快的结论不是纯扯淡吗



> mpv给第三方的下载地址都写免责声明呢，你这自信是哪来的。。。

第一次玩MPV,以后要注意了,感谢提醒!

**Reply by hooke007** [reply]
重点是mpv吗。。。本身release的东西就可以和源代码毫无关联
我吐槽的是“来路很正”这种莫名奇妙的观点，你不逆向又怎么知道来路很正。。

**Reply by leezm** [reply]
> 重点是mpv吗。。。本身release的东西就可以和源代码毫无关联 我吐槽的是“来路很正”这种莫名奇妙的观点，你不逆向又怎么知道来路很正。。

一般活跃项目默认来路问题咯

**Comment by hooke007** [comment]
开一个公共维基计划，见一层的内容更新 https://github.com/hooke007/MPV_lazy/discussions/495#discussion-7087940
任何人觉得比较闲都可以编辑，目的并不是为了复制官方wiki的内容，毕竟很多脚本小众或不实用。
我打算有时间把一些看起来有用的分类贴上去，其它人有自建的脚本（特别是针对中文本地化作用的脚本）也随意补充

**Comment by a82263** [comment]
双击应用程序没反应怎么回事

**Reply by hooke007** [reply]
那你是不是至少写一下什么硬件什么平台，不然让别人给你掐指一算呢

**Reply by a82263** [reply]
安装在桌面

**Reply by hooke007** [reply]
...意思是你电脑是什么配置

**Comment by HunterPR0** [comment]
不知什么原因，播放中途会自动闪退

**Reply by hooke007** [reply]
>不知什么原因

那你只有自己去看系统崩溃记录了。。。

**Reply by HunterPR0** [reply]
> > 不知什么原因
> 
> 那你只有自己去看系统崩溃记录了。。。

看过了，应用程序的崩溃日志里找不到记录

**Comment by mengyvleihen** [comment]
发现一个很奇怪的问题，2024V2版本在播放部分X265的视频时存在莫名其妙的卡顿及音画不同步，而2024V1反而没有这个问题。
有问题的视频bt链接rarbg的环太平洋2：magnet:?xt=urn:btih:3AD0CD21D89B69FE391B984F2F5BB4EACA77FE3F。
这个问题在rarbg的1080p x265视频中尤其多，4k分辨率的反而没有问题。

**Reply by hooke007** [reply]
看一楼

**Reply by mengyvleihen** [reply]
哦哦哦，谢谢大佬。

**Comment by acu754** [comment]
大佬，请问超分和插帧不能一起开吗

**Reply by hooke007** [reply]
为什么不能

**Reply by acu754** [reply]
我开了插帧，再开超分，它自动关了我的插帧，反之同理。我是伪UI那里开的

**Reply by hooke007** [reply]
因为预设里就是没考虑并行多个滤镜。手动自己去改命令
https://github.com/hooke007/MPV_lazy/blob/d16afd57f72f57c9f9bbf504ab43f435ec06b804/portable_config/input_list.conf#L44

**Reply by hooke007** [reply]
或者看文档改脚本 https://github.com/hooke007/MPV_lazy/discussions/313#discussioncomment-5968181

**Comment by newyorker123** [comment]
大佬好，请问一下，更新记录里写uosc 增加了 “次字幕轨列表”。但我使用的时候怎么都找不到这个功能。请问怎么使用uosc的次字幕功能呢？

**Reply by hooke007** [reply]
`script-binding uosc/subtitles-sec`

**Reply by newyorker123** [reply]
感谢大佬！

**Comment by hoousun** [comment]
用鼠标左键选中 记事本 里的 文本链接，然后拖拽到播放器界面上，没有任何反应，控制台上也没有信息；但拖拽浏览器地址栏上的链接，则可以播放。请问，有办法拖拽记事本的文本链接到播放器上直接播放吗？像 mpv.net 般

**Comment by jtky233** [comment]
大佬您好，关于RTX HDR我已经 使用官网的最新构建11.7日版本替换了。
结合您给出的 
gpu-context = d3d11
hwdec = d3d11va          
 d3d11-output-csp = srgb          
 vf-append = d3d11vpp=format=nv12:nvidia-true-hdr （改成也试过nvidia-true-hdr=yes）
以及翻阅周边文章， 调整色彩格式等等......都无法无法获得类似potplayer + mpc 的hdr效果，只能让画面变成比sdr亮的结果。
我使用的是win11 23h2， 驱动566.03。
想知道我是否丢掉了一些命令，或者的确是驱动或一些情况,， 目前效果只能 控制面板不显示活动 rtx hdr的效果也只能让画面变亮。
顺带感谢大佬制作的nv补帧包，效果和流畅度都比同类， 不知道高到哪里去了。
若我能进一步启用RTX HDR 那就完美了

**Reply by hooke007** [reply]
windows的hdr开关打开了吗，说实话我也无法帮助，windows的设备驱动太复杂不统一，也有人说无效

**Reply by jtky233** [reply]
当然开启了，也换了一些其他版本都无法使用。
 不过已经放弃折腾，专心享受观影了。 一个mpv看高帧一个pot+mpc 看hdr了

**Comment by tumibloom** [comment]
![image](https://github.com/user-attachments/assets/1b41b37e-5af1-4e35-874e-72bb1d156964)
这是mpv-lazy-2024V2标准版，但是字幕发虚，不够清晰；
![image](https://github.com/user-attachments/assets/6bfc4718-9f7e-494f-a24f-c07153d176c5)
这是mpvnet版本的，我从网上找的资源，但是字幕显示是可以的，有什么办法可以修复一下吗

**Reply by hooke007** [reply]
一个过时版本的截图没有意义，给我字幕来复现

**Reply by hooke007** [reply]
不管是哪个mpv的字幕都是原始的功能实现，区别就是版本或者参数不一样

**Reply by tumibloom** [reply]
[Dark.S02E01.2160p.NF.WEBRip.DDP5.1.x264-NTb.zip](https://github.com/user-attachments/files/17691268/Dark.S02E01.2160p.NF.WEBRip.DDP5.1.x264-NTb.zip)
问题所见在4分23秒

**Reply by hooke007** [reply]
预期之内，这个参数的副作用
https://github.com/hooke007/MPV_lazy/blob/b6b10a53fe86297ed0eef555172331b24bedcd08/portable_config/mpv.conf#L161

**Comment by Kirishim-arashi** [comment]
解压出错 这是我的问题吗 我看了sha256也没问题呀 奇怪
![image](https://github.com/user-attachments/assets/b3f4e06e-6225-4519-967f-99c979243d67)

**Reply by hooke007** [reply]
换个新点的解压内核吧

**Comment by rplalala** [comment]
![{B202A52D-2834-4d75-8D45-874370E429D0}](https://github.com/user-attachments/assets/80f1bc33-6f1a-4754-8d1f-cb471ba05b91) 请问这个未知的着色器是什么

**Reply by hooke007** [reply]
gpu-next的已知问题

**Reply by rplalala** [reply]
> gpu-next的已知问题

感谢，请问 profiles 里有办法写 让 某个着色器触发时 改变某个属性吗。 比如 在打开 adaptive_sharpen_RT.glsl 时 把 sigmoid-upscaling 设置为 no

**Reply by hooke007** [reply]
https://mpv.io/manual/master/#command-interface-vo-passes 在这个属性里查着色器pass里的对应关键字作为触发条件

**Reply by rplalala** [reply]
> https://mpv.io/manual/master/#command-interface-vo-passes 在这个属性里查着色器pass里的对应关键字作为触发条件

感谢已经实现了，我是之前版本更新的，我以前下的版本 sigmoid-upscaling 默认是yes ，请问为什么这版本要改为no，我怕开了后有影响。是因为默认打开的这个着色器的原因吗？AMD_FSR_EASU_rgb_RT.glsl

**Reply by hooke007** [reply]
感知不强，没有必要

**Reply by verygoodlee** [reply]
> gpu-next的已知问题

强迫症看着是有点难受，目前看修复的希望不大，说是API要大改才能解决
https://code.videolan.org/videolan/libplacebo/-/issues/166

工作在亮度平面的着色器好像没这个问题，我日常就使用的亮度平面放大的FSR，
反正也碰不到几次RGB源的视频，图片倒是RGB居多，但是也不用mpv看图片

**Comment by soleige** [comment]
[mpv-lazy.log](https://github.com/user-attachments/files/17789424/mpv-lazy.log)
今天从24V0换到24V2试试水，rife4.25lite+ani4kv2 artcnn一起开突然崩溃了一次，之后就无法正常启用着色器了

**Reply by hooke007** [reply]
日志里没看到着色器没启用，正常启用

**Reply by soleige** [reply]
![image](https://github.com/user-attachments/assets/858e3eee-7844-4b7e-943f-d22b66e5ffa5)
但是着色器列表里面没变化

**Reply by hooke007** [reply]
![image](https://github.com/user-attachments/assets/95993fda-28a6-4989-9fb7-36d8bc97d466)
你窗口分辨率都不满足原片尺寸1.2倍，当然不触发

**Reply by hooke007** [reply]
你先看看着色器的说明吧，虽然没写这个，但是一些基本的通用知识还是需要的

**Reply by soleige** [reply]
原来是这样……24V0稳定用了太久，以前记住的都忘了

**Reply by hooke007** [reply]
而且我更新说明也写了
![image](https://github.com/user-attachments/assets/6af2419c-32d7-455d-8778-0797e408f9b1)

**Reply by soleige** [reply]
尝试了下ani4kv2 aniSD的glsl，感觉好吃性能，比rife4.25lite 1080p 2x 的开销都高（以显卡功耗为判断标准）

**Reply by soleige** [reply]
大佬，打开视频 全屏- 挂载artcnn c4f32，MPV会立刻闪退，是单纯的性能不够吗？
[mpv-lazy.log](https://github.com/user-attachments/files/17790531/mpv-lazy.log)

**Reply by hooke007** [reply]
没遇到过

**Reply by soleige** [reply]
找到原因了，删掉portable_config下的_cache目录，重新生成shader cache就正常了，使用ArtCNN_C4F32_CMP.glsl第一次生成cache不知道为什么很慢，CPU占用率只有10%左右，我以为卡死了就结束进程了，如果耐心等完它生成cache然后继续播放就正常

刚打完字很奇怪的又出现了一次闪退，再次删除shader cache解决

**Reply by soleige** [reply]
大佬，除了在input_uosc.conf里面添加（习惯的键位加满了，再加的话按着别扭），有什么可以快速更换glsl的方法吗，如果按~呼出控制台输入 change-list glsl-shaders toggle "~~/shaders/CfL_Prediction.glsl" ，除了第一次要复制粘贴麻烦点（手打好长），后面倒是可以按↑仅修改文件名，但是每次退出进程后不会保存控制台输入记录

**Reply by hooke007** [reply]
键位怎么可能用得完，连键还不够你用吗，你去看看input conf的基本说明

**Reply by soleige** [reply]
大佬，跑分模式下测试24FPS的文件，vf=vapoursynth=rife2x, display-fps-override要填的源帧率应该是48？

**Reply by hooke007** [reply]
是

**Comment by Darkvanci** [comment]
你好 播放列表默认文件夹可以自定义吗 没有找到相关教程 config也翻了一遍没有找到
![1](https://github.com/user-attachments/assets/5c320059-af29-4049-8731-2f1000a0f80b)

**Reply by hooke007** [reply]
那这是什么
https://github.com/hooke007/MPV_lazy/blob/b6b10a53fe86297ed0eef555172331b24bedcd08/portable_config/script-opts.conf#L88

**Reply by Darkvanci** [reply]
好的谢谢

**Comment by w-calvin** [comment]
更新是需要uninstall取消注册再安装新版吗
因为release界面说了不能覆盖，也没有在discussion搜到

**Reply by hooke007** [reply]
主程序位置没变就不需要重新注册

**Comment by yanghaoynu** [comment]
首先感谢作者，很感激，让我找到了
我电脑配置如下图：
<img width="306" alt="显卡" src="https://github.com/user-attachments/assets/8b65bfc6-b13d-44d8-ae8a-0a4a68765701">
<img width="1516" alt="系统配置" src="https://github.com/user-attachments/assets/0723e223-7ef5-45bf-bde5-2a186d28e897">

（一）一方面分享使用心得：我在不同的解码器之间来回折腾了几天，最后选定了的硬解模式还是如下：
vo = gpu-next
gpu-api = d3d11
gpu-context = d3d11
hwdec = dxva2
profile = HDR_generic
hwdec-codecs = all
d3d11-output-csp=pq
d3d11-output-format=rgb10_a2
target-trc=pq  
target-prim=bt.2020                        
vf=format:dolbyvision=yes
4k的10g左右的视频，内存大概300m，cpu大概1%
以上配置模式，色彩比win11自带的“媒体播放器”还要惊艳，其实win11自带的媒体播放器效果也是非常好的，我估计微软自己有一套api。
其他的硬解模式在播放部分mp4的时候，我总觉得温度好高，内存也是1g往上，后来我打开任务管理器，cpu和内存使用是很低，但是我的独显差不多被占用完了，不注意看根本不知道。因为篇幅的原因，如果作者需要我还原，我下次来重新发图。
（二）想请教作者，如何去除视频的上下黑边
<img width="17" alt="黑边" src="https://github.com/user-attachments/assets/21058876-e380-45f7-98cb-b8e3675aad05">

（三）请教作者，这个警告是我因为我配置不对还是本身视频的问题
<img width="1598" alt="mkvhdr警告信息" src="https://github.com/user-attachments/assets/b63d3cba-a24f-4fc6-bc3f-3dc6b2d1412b">

测试视频：通过百度网盘分享的文件：Banding TEST-Bulgaria 4K HDR 60P.mkv
链接：https://pan.baidu.com/s/1B_Q5_7jhmWdbvURob4YD9w 
提取码：1234 
--来自百度网盘超级会员V5的分享

配置文件：
[配置文件.txt](https://github.com/user-attachments/files/17879124/default.txt)
（四）算是建议：配置文件中配置#d3d11-adapter和通过操作系统指定播放器使用独显的机制是完全不一样的，前者在信息输出里面会报错，通过系统指定mpv使用的显卡解决的问题。
谢谢作者解答！！！我也自己查了资料了，没解决掉。

**Reply by yanghaoynu** [reply]
黑边的图因为网络问题没成功上传，这里补上：
<img width="17" alt="黑边" src="https://github.com/user-attachments/assets/114a340a-0852-4cde-bd24-c425ec468a72">

**Reply by hooke007** [reply]
dxva的解码颜色有问题。
vf=format:dolbyvision=yes没有用，因为是默认值。
解封装报告的容器问题。
黑边 panscan

**Reply by yanghaoynu** [reply]
<img width="1600" alt="黑边" src="https://github.com/user-attachments/assets/a76b4252-5b61-46ff-87ac-d636e817d56b">

**Reply by yanghaoynu** [reply]
@hooke007 
<img width="1437" alt="黑边" src="https://github.com/user-attachments/assets/832a6d5d-15f8-4715-b357-011ab43768d2">

**Reply by hooke007** [reply]
黑边 panscan
我说过了

**Reply by yanghaoynu** [reply]
完美：panscan=1，谢谢作者。

**Reply by yanghaoynu** [reply]
去黑边效果：<img width="1600" alt="panscan=1效果" src="https://github.com/user-attachments/assets/b906f0b9-5733-4ef0-812f-b781c96e3e18">赞啊，我找了好几个小时了。

**Comment by yanghaoynu** [comment]
启用d3d11解码器后，显卡百分之百的问题复原了，请作者看看。
<img width="832" alt="conf文件" src="https://github.com/user-attachments/assets/fd9d20f8-7bb8-45e5-8792-6dd5cecb1800">
<img width="1048" alt="视频信息" src="https://github.com/user-attachments/assets/e32dec75-16bb-44c7-b026-a44a2020baa8">
<img width="1600" alt="gpu100%" src="https://github.com/user-attachments/assets/eca9fa0d-4ee7-4e95-aa4e-20e1cd221c3c">

**Reply by yanghaoynu** [reply]
<img width="832" alt="conf文件" src="https://github.com/user-attachments/assets/2e3de06f-e810-46ba-b3f9-04edbb8861f7">
<img width="1600" alt="gpu100%" src="https://github.com/user-attachments/assets/d49826cb-cd27-4326-88bf-5079b1ee02c6">
<img width="1048" alt="视频信息" src="https://github.com/user-attachments/assets/52f85699-060d-4dbb-aa80-f8d5b070b1c6">

**Reply by hooke007** [reply]
无法复现，我也没有同样硬件复现

**Reply by yanghaoynu** [reply]
作者您好：非daxa2下的显卡占比100%问题得到解决或者临时解决，我禁用了script-opts.conf中的全局属性保存恢复。
<img width="1600" alt="解决窗口最大化以后非daxa2解码导致显卡占比100%和主板温度过高的问题" src="https://github.com/user-attachments/assets/39b873e8-931f-431f-aecb-a55e4db693ba">

**Reply by hooke007** [reply]
它恢复也只是恢复了着色器，看不出来有什么关联

**Reply by hooke007** [reply]
如果是在恢复着色器产生的问题，说明问题在着色器的应用步骤中。给我看截图没有意义，还不如日志。

**Reply by yanghaoynu** [reply]
作者您好：
我不知道为啥，是不是清除了mvp内部的着色器、缓存文件夹，我只能ctrl+1手动还原问题。这是日志文件：
[mpv-lazy.log](https://github.com/user-attachments/files/17897055/mpv-lazy.log)

**Reply by hooke007** [reply]
mpv一直在尝试编译着色器，你的目录真的有读写权限吗？你的stats里真的有显示着色器在运行吗

**Reply by hooke007** [reply]
stats不是日志

**Reply by yanghaoynu** [reply]
解码器d3d11va：操作顺序，打开视频--ctrl+1（此时gpu正常）--窗口最大化（gpu100%且不下降），有且只有dxxa2无此问题。
<img width="962" alt="默认窗口" src="https://github.com/user-attachments/assets/ef534bf3-7d2f-469d-8f44-038cfbd0538f">

<img width="1600" alt="窗口最大化" src="https://github.com/user-attachments/assets/4ba9e934-ac2b-4c4c-878b-c13b28fcb7d2">

**Reply by hooke007** [reply]
着色器的stats在第二页。。。

**Reply by yanghaoynu** [reply]
怎么翻页哦，up和down无效。是不是只有一页。

**Reply by hooke007** [reply]
数字键

**Reply by yanghaoynu** [reply]
<img width="1600" alt="着色器" src="https://github.com/user-attachments/assets/f3e14d52-3cee-42ee-9dec-e917f7e7019e">

**Reply by hooke007** [reply]
dxva2的图呢

**Reply by yanghaoynu** [reply]
<img width="1600" alt="daxa2着色器" src="https://github.com/user-attachments/assets/f7ce5458-2bde-4dbc-8d81-dbe107bc787b">
确实少很多，只有默认配置文件袋那个AMD_FSR_EASU_rgb_RT.glsl，是不是daxa2不支持，操作过程一样的。

**Reply by hooke007** [reply]
没显示说明实际没激活。

**Reply by hooke007** [reply]
所以显卡占用问题可能很简单，你的硬件性能支撑不起这个着色器。

**Reply by verygoodlee** [reply]
帧生成时间平均45ms, 1000/45=22.2 , 24帧都撑不住更别说60帧了，你这看的也不是动画，ArtCNN系列的着色器只适用于动画，非动画开了反而会影响画质
![image](https://github.com/user-attachments/assets/5ed03be1-a954-4c8e-a32d-8ab2f0869a72)

**Reply by yanghaoynu** [reply]
谢谢指正。是否我配置文件哪个参数本身设置错了，还是说不开ArtCNN就好。

**Comment by HeatCaCO3** [comment]
srt字幕默认颜色是橙色的，该如何改变色
![无标题 ](https://github.com/user-attachments/assets/bc1171a8-5bbc-4e02-8bba-d6a1530d72f1)

**Reply by hooke007** [reply]
至少花点时间看看仓库里mpvconf写的东西

**Reply by yanghaoynu** [reply]
在mpv.conf中有具体的配置信息，找到字幕部分，手动测试，不知道颜色的话可以用画图工具取色。

**Comment by yanghaoynu** [comment]
新开一楼，咨询作者：要实现以下功能，补丁包里面是否有相应的插件。
<img width="1600" alt="投到设备播放" src="https://github.com/user-attachments/assets/86a264cd-cdaf-4e06-b2b6-03afa56f89a3">

**Reply by zhongfly** [reply]
没有，你自己在仓库里搜下就知道。
如果你要问怎么实现，谷歌搜一下/Github内搜索相应的关键词

**Comment by LeiBuSi-Ultra** [comment]
大佬，无意间发现播放速度有点过快（有用秒表对比过）

**Reply by hooke007** [reply]
display-resample会调速

**Comment by Yashiro410** [comment]
请问更新MPV版本后缩略图无法正常显示该如何解决？
补充1.更新方式为下载新版MPV压缩包后覆盖解压
补充2.进度条缩略图显示为黑色

**Reply by Yashiro410** [reply]
补充3.我下载标准版时自行更新就出现了此问题，查阅文档后以为是自编译mpv的原因，就下载了noBin版本重试，仍是相同情况

**Reply by Yashiro410** [reply]
补充4.下载noBin版后我使用NV包内的文件覆盖解压了原文件夹，再进行的补充1步骤

**Reply by hooke007** [reply]
你有能正常用的版本吗

**Reply by hooke007** [reply]
修了 https://github.com/hooke007/MPV_lazy/commit/0864512d734351a91443fced0c1cf08943293af8 ，应该是因为mpv最近改动了vo的初始化逻辑

**Comment by yanghaoynu** [comment]
终于找到了file_brower读取本地分区错误的原因了。注意：这个错误目前只在win11 24h2中存在。然后打开本地资源也会后台报错，原因暂时我分析是因为win11 24h2下是没有pwsh命令了，除非你手动再去修改path。
1、win11 24h2彻底弃用了wmic功能，除非你手动添加，winroot.lua是使用wmic命令获取本地磁盘信息的，如图：
<img width="905" alt="Snipaste_2024-12-13_02-16-38" src="https://github.com/user-attachments/assets/af50ef51-c9e2-4a37-9c6f-008a90e63091" />
查看wmic被弃用的信息：
https://www.sysgeek.cn/wmic-to-powershell-migration/
https://www.ithome.com/0/781/543.htm
wmic的安装在设置--系统--可选功能里面。
2、mpv通过ctrl+o打开本地资源管理器会报错，是因为win11 24h2默认不支持pwsh命令，而脚本源码中又使用了该命令，如图：
<img width="1097" alt="Snipaste_2024-12-13_02-17-59" src="https://github.com/user-attachments/assets/cf9985da-5dd8-4296-b3a3-0e0730b09743" />
3、正常按下o键应该如下：
<img width="953" alt="Snipaste_2024-12-13_02-18-46" src="https://github.com/user-attachments/assets/010e978e-d773-4f4b-9917-af1b34d1c1ba" />
在此，跟大家共享经验：

**Comment by CrayonSSR** [comment]
### 大佬，字幕的尺寸缩放是不是失效了

#### 我把之前的字幕缩放配置，粘贴到了`input_uosc.conf`
```
 Z                 add sub-scale -0.1                                    #! 播放 > 字幕字号 -0.1
 X                 add sub-scale  0.1                                    #! 播放 > 字幕字号 +0.1
```

#### 结果是屏幕左上角有提示，但实际不会对字幕产生任何作用
![image](https://github.com/user-attachments/assets/0299aa59-d92a-48aa-9498-4a150b1ac2cc)

#### 字幕文件&配置文件
- [subtitle.zip](https://github.com/user-attachments/files/18121950/subtitle.zip)
- [portable_config.zip](https://github.com/user-attachments/files/18121948/portable_config.zip)

**Reply by hooke007** [reply]
犯懒了，先提个discussion看看 https://github.com/mpv-player/mpv/discussions/15509

**Reply by hooke007** [reply]
https://mpv.io/manual/master/#options-sub-scale-signs

**Comment by tumibloom** [comment]
![image](https://github.com/user-attachments/assets/004bbb64-ff3d-4357-8f81-badee1d6bb5b)
这个统计信息的字体大小可以调整吗，太大了看不全，我在` script-opts.conf `里面调整但是好像没用

![image](https://github.com/user-attachments/assets/e8b26c01-fa08-4dc9-b028-2db37585f79b)
这个进度条能给改没吗，我找到了` script-opts-append = uosc-timeline_style=line ` 但是改动的不是这个

**Reply by hooke007** [reply]
这是你自己用的？和懒人包有啥关系

**Reply by hooke007** [reply]
https://github.com/hooke007/MPV_lazy/blob/3ee6c2ff5c22449ff3c5b252c4e6dd7f75f6fed4/portable_config/script-opts.conf#L187
不是很明白没用是什么意思

https://mpv.io/manual/master/#options-osd-on-seek

**Reply by tumibloom** [reply]
> 这是你自己用的？和懒人包有啥关系

这个就是我从项目里复制到配置里之后的效果，那个字体大小现在可以生效了，最开始直接解压是和懒人包一样的，但是后面为了更好的改，就直接把仓库里的` mpv.conf` 复制到本地了（include也改成对应的本地文件目录了），然后就有那个进度条的组件了
